### PR TITLE
[script.simkl] 3.0.0

### PR DESCRIPTION
**Pull request by ekleop**
### Description
Added support for Kodi 19, same functionality as previous version in Jarvis push.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.simkl/addon.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="3.0.0"
+       provider-name="David Davó, Simkl">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.service" library="service.py" start="login" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Simkl Scrobbler with AI file detection system</summary>
```
Comment (enen92) [inline script.simkl/addon.xml:11]:
```suggestion
    <summary lang="en_GB">Simkl Scrobbler with AI file detection system</summary>
```

File: script.simkl/addon.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="3.0.0"
+       provider-name="David Davó, Simkl">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.service" library="service.py" start="login" />
```
Comment (enen92) [inline script.simkl/addon.xml:9]:
```suggestion
  <extension point="xbmc.service" library="service.py" />
```

File: script.simkl/addon.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Simkl Scrobbler with AI file detection system</summary>
+    <summary lang="en_US">Simkl Scrobbler with AI file detection system</summary>
+    <description lang="en">Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into. Sign up at https://simkl.com to get all of these features: &#10;&#10;- Automatically track what you’re watching.&#10;- Receive notifications about new episodes.&#10;- Find recommendations based on your watch history.&#10;- See your friend's ratings.&#10;&#10;Simkl was created to help people, like you, to keep track of what you watch and receive notifications about new episodes and new movie releases based on your preferences and watch history.</description>
```
Comment (enen92) [inline script.simkl/addon.xml:13]:
```suggestion
    <description lang="en_GB">Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into. Sign up at https://simkl.com to get all of these features: &#10;&#10;- Automatically track what you’re watching.&#10;- Receive notifications about new episodes.&#10;- Find recommendations based on your watch history.&#10;- See your friend's ratings.&#10;&#10;Simkl was created to help people, like you, to keep track of what you watch and receive notifications about new episodes and new movie releases based on your preferences and watch history.</description>
```

File: script.simkl/addon.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    <description lang="en_US">Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into. Sign up at https://simkl.com to get all of these features: &#10;&#10;- Automatically track what you’re watching.&#10;- Receive notifications about new episodes.&#10;- Find recommendations based on your watch history.&#10;- See your friend's ratings.&#10;&#10;Simkl was created to help people, like you, to keep track of what you watch and receive notifications about new episodes and new movie releases based on your preferences and watch history.</description>
+    <disclaimer lang="en">Not tested on Android</disclaimer>
+    <platform>all</platform>
+    <license>GPLv3</license>
```
Comment (enen92) [inline script.simkl/addon.xml:17]:
```suggestion
    <license>GPL-3.0-only</license>
```

or

```suggestion
    <license>GPL-3.0-or-later</license>
```

File: script.simkl/service.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+from resources.lib.utils import log
+from resources.lib.utils import system_lock
+
+from resources.lib import engine
+from resources.lib import api_simkl
+from resources.lib import events
+
+if __name__ == "__main__":
+    system_lock("SimklTrackerRun", 5)
+    log("dir = " + str(xbmc.translatePath("special://home")))
```
Comment (enen92) [inline script.simkl/service.py:20]:
```suggestion
    log("dir = " + str(xbmcvfs.translatePath("special://home")))
```

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments

**Comment by ekleop** [comment]
Everything was updated according your guide, thanks

**Review by enen92 — APPROVED** [review]

**Pull request merged**